apps/dashboard/package.json

```diff
@@ -6,7 +6,7 @@
     "@popperjs/core": "^2.11.8",
     "@rails/ujs": "^7.0.1",
     "@uppy/core": "^4.0",
-    "@uppy/dashboard": "4.3.4",
+    "@uppy/dashboard": "^4.0",
```

Comment:
I made this change because previously I'd locked it to `4.3.4` to upgrade other things while keeping this version the same. That clearly didn't work, so we may as well just have all `@uppy` packages lock to `^4.0`.